src/garage/torch/algos/vpg.py

```diff
@@ -61,6 +64,7 @@ def __init__(
             center_adv=True,
             positive_adv=False,
             optimizer=None,
+            optimizer_args=None,
```

Comment:
i'm really not a fan of this pattern of passing dicts of args for constructors into other constructors. can we somehow construct-and-pass the optimizer instead? or just flatten these args into the parent constructor?

@krzentner your thoughts?